docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+* `Share results of experiments with your team <https://docs.neptune.ai/getting-started/quick-starts/how-to-share-results.html#use-cases-share-results-with-team>`_
+
+* `Clean up your Jupyter workflow without git <https://docs.neptune.ai/getting-started/quick-starts/how-to-clean-up-jupyter.html#use-cases-clean-jupyter-workflow>`_
+
+In this tutorial, you'll build a simple spam classifier using Keras. The task is a binary classification problem,\
+as such you'll create a neural network that takes spam messages as input, and then return a prediction of 1 (spam) or 0 (not-spam). \
+The |SPAM| dataset contains 5572 SMS messages and a label. Using this dataset, you are going to create a machine learning model that learns to detect "spam" or not "spam".
+
+.. |SPAM| raw:: html
+
+    <a href="https://www.kaggle.com/uciml/sms-spam-collection-dataset" target="_blank">SPAM</a>
+
+What will you get after
+-----------------------
+At the end of this tutorial, you'll have a created an accurate model that can effectively predict if an SMS to you is Spam or not, and you'll have effectively tracked and saved all experimentation details including metrics, artifacts like the trained model, plots/charts showing visualization, training and testing datasets, as well as your experimentation notebooks.
+Links:
+|source code| |open in colab|
+
+Prerequisites:
+--------------
+* Have working knowledge of Python
+* Have basic understanding of machine and deep learning
+* Have basic understanding of Keras framework
+* Have used or created a neural network before
+* Have a neptune account and API token. You can just |signup| here to get one
+
+If you tick all the prerequisites listed above, then you're ready to move on to the next part of this tutorial.
+
+Setting up your environment
+---------------------------
+Head over to |colab| and create a new notebook in order to get started. Note that this tutorial can be run locally with minimal changes, you just have to install each package you'll be using in your local environment. If you have installed a distribution like Anaconda, then you have almost all the packages you need except tensorflow keras and neptune. 
+
+I'll walk you through installation and setting up of neptune. In order to install Tensorflow Keras, you can visit this link.
+
+Neptune does not come preinstalled on colab, and also needs to be initialized with an API token before you can log metrics. In the steps below, you're going to create a new Neptune project, install a Neptune client on colab via pip, and also initialize the project with your API token. Follow the steps below to achieve this:
+
+
+Step 1: If you have signed-up, login into your account and create a new project. In the new project settings, add a project name (spam-classifier), copy the initialization code, you can set your project to either public or private, and also add a description. Finally click apply to save the new project.
```

Comment:
People don't have to create a new project to use this integration -> we can link to a section about creating a project but not write about it here. 

Also note/remember that you can use the `api_token=ANONYNOUS` and project shared/keras-integration to log those things without registering even.